### ISSUE TITLE
cli import error

### ISSUE THREAD

**Issue by bnelson-czi** [napari/npe2#141]
* npe2 version: 0.2.1
* Python version: 3.9.11
* Operating System: macOS 12.3

### Description

Working in a clean conda environment, @nclack and I ran into an issue running `npe2 validate .` on a test plugin

### What I Did
```
(plugin) czirwc1macos2994:napari-fits-reader bnelson$ npe2 validate .
Traceback (most recent call last):
  File "/Users/bnelson/anaconda3/envs/plugin/bin/npe2", line 5, in <module>
    from npe2.cli import main
  File "/Users/bnelson/anaconda3/envs/plugin/lib/python3.9/site-packages/npe2/cli.py", line 6, in <module>
    import typer
  File "/Users/bnelson/anaconda3/envs/plugin/lib/python3.9/site-packages/typer/__init__.py", line 12, in <module>
    from click.termui import get_terminal_size as get_terminal_size
ImportError: cannot import name 'get_terminal_size' from 'click.termui' (/Users/bnelson/anaconda3/envs/plugin/lib/python3.9/site-packages/click/termui.py)
```

**Comment by tlambert03** [comment]
this is an upstream bug in typer: https://github.com/tiangolo/typer/issues/377

you can fix locally by downgrading click: `pip install 'click<8.1.0'` ...

note also that `npe2 validate` takes a package name or a manifest yaml file  (so `'.'` won't work)

**Comment by tiangolo** [comment]
I just released Typer `0.4.1` that should handle this. :rocket: :nerd_face:

**Comment by tlambert03** [comment]
Thanks @tiangolo!
